src/util/extent.js

```diff
@@ -128,9 +128,9 @@ export default function() {
 
     extents.include = function(x) {
         if (!arguments.length) {
-            return extraPoint;
+            return extraPoints;
         }
-        extraPoint = x;
+        extraPoints = x;
```

Comment:
I don't think any of the components do or should support additive operations. It used to be the case that data.random.financial had mutable internal state but that's now been encapsulated in the stream instance.  

Can you give a common example where it would be useful to have the additive support?